src/components/pages/ImageClassification/ImageClassificationTable/ImageClassificationObservationTable.jsx

```diff
@@ -449,6 +449,18 @@ const ImageClassificationObservationTable = ({
                             IMAGE_CLASSIFICATION_STATUS_LABEL[file.classification_status?.status]
                           )}
                         </StyledTd>
+                        <StyledTd textAlign="center">
+                          <MuiTooltip
+                            title={
+                              language.imageClassification.imageClassificationModal.tooltip
+                                .removePhoto
+                            }
+                          >
+                            <ButtonCaution type="button" onClick={() => openRemovePhotoModal(file)}>
+                              <IconClose aria-label="close" />
+                            </ButtonCaution>
+                          </MuiTooltip>
+                        </StyledTd>
                       </Tr>
                     )
                   }
```

Comment:
@hurrendor  This is the only change, The others are autoformating when I use git commit.